packages/server/src/tests/banners/abandonedBasketTests.ts

```diff
@@ -0,0 +1,49 @@
+import { designableBanner } from '@sdc/shared/src/config/modules';
+import type { BannerTest, BannerTestGenerator, BannerVariant } from '@sdc/shared/types';
+
+const baseAbandonedBasketTest: Omit<BannerTest, 'name' | 'variants'> = {
+    bannerChannel: 'abandonedBasket',
+    isHardcoded: true,
+    userCohort: 'Everyone',
+    // We can use this as the feature switch
+    status: 'Draft',
+    priority: 99,
+    locations: [],
+    contextTargeting: { tagIds: [], sectionIds: [], excludedTagIds: [], excludedSectionIds: [] },
+};
+
+const baseAbandonedBasketVariant: Omit<BannerVariant, 'bannerContent'> = {
+    name: 'control',
+    modulePathBuilder: designableBanner.endpointPathBuilder,
+    // Requires this design to exist in the RRCP!
+    template: { designName: 'ABANDONEDBASKET' },
```

Comment:
If this design doesn't exist:

On the server side, the `design` field in the props will be `undefined`: https://github.com/guardian/support-dotcom-components/blob/main/packages/server/src/tests/banners/channelBannerTests.ts#L60

And on the client side, it'll silently not render the banner:
https://github.com/guardian/support-dotcom-components/blob/main/packages/modules/src/modules/banners/designableBanner/DesignableBanner.tsx#L111

Perhaps we should have monitoring for missing designs, but I think that's beyond the scope of this PR